src/shared/read-agents.c

```diff
@@ -596,6 +596,10 @@ void delete_sqlite(const char *id, const char *name)
 /* Delete diff folders */
 void delete_diff(const char *name)
 {
+    if (!name || *name == '\0') {
```

Comment:
Coding style only allow using negation operator on booleans
```suggestion
    if (NULL == name || *name == '\0') {
```

Reply:
Fixed !